README.md

```diff
@@ -37,10 +37,8 @@ Options:
       --hosted-url      The server that hosts <package>.
                         (defaults to "https://pub.dev")
   -l, --line-length     The line length to use with dartfmt.
-      --verbosity       Configure the details in the output.
-                        [compact, normal (default), verbose]
-      --[no-]scores     Include scores in the output JSON.
-      --[no-]report     Print markdown report instead of JSON.
+      --[no-]hosted     Shortcut to `--source hosted`.
+      --[no-]verbose    Print the full JSON output instead of the markdown report.
```

Comment:
Should we call this `--json` or `--machine`?

Reply:
We already have `--json` for the logs... which should be probably reused for this too (if you want json logs, why would you want non-json reports)?